prototype/mixins.py

```diff
@@ -29,3 +31,34 @@ class SlugFromKwargsMixin():
     @property
     def slug(self):
         return self.kwargs['slug']
+
+
+class InternationalNewsCMSPageMixin(GetCMSPageByFullPathMixin):
+
+    @cached_property
+    def page(self):
+        response = cms_api_client.lookup_by_full_path(
+            full_path=prefix_international_news_url(self.request.path),
```

Comment:
you could make this DRYer by adding a property to `GetCMSPageByFullPathMixin` called `cms_lookup_path` that returns `self.request.path`:

```
class GetCMSPageByFullPathMixin(TemplateView):
     @property
     def cms_lookup_path(self):
        return self.request.path

    @cached_property
    def page(self):
        response = cms_api_client.lookup_by_full_path(
            full_path=self.cms_lookup_path,
            draft_token=self.request.GET.get('draft_token'),
        )
        return handle_cms_response(response)
```
then you can do
```
class InternationalNewsCMSPageMixin
    @property
    def cms_lookup_path(self):
        return prefix_international_news_url(self.request.path)
```

that has less duplication of logic :)

Reply:
ty i was wondering how i could improve that